### PR TITLE
docs(localization): translate client-control from EN into ZH

### PR DESCRIPTION
**Pull request by Xunzhuo**
### What this PR does / why we need it:

translate client-control from EN into ZH

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/plugins/client-control.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+- [**名称**](#名称)
+- [**属性**](#属性)
+- [**如何启用**](#如何启用)
+- [**测试插件**](#测试插件)
+- [**禁用插件**](#禁用插件)
+
+## 名称
+
+`client-control` 插件能够动态的控制 Nginx 的行为去处理客户端的请求。
```
Comment (spacewander) [inline docs/zh/latest/plugins/client-control.md:34]:
```suggestion
`client-control` 插件能够动态地控制 Nginx 处理客户端的请求的行为。
```

Comment (Xunzhuo) [reply docs/zh/latest/plugins/client-control.md]: Done. Thanks.

**Review by spacewander — APPROVED** [review]

File: docs/zh/latest/plugins/client-control.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+| max_body_size | integer        | 可选    |              | >= 0 | 动态设置 `client_max_body_size` 的大小 |
+
+## 如何启用
+
+以下是一个示例，在指定路由中启用插件：
+
+```shell
+curl -i http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/index.html",
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/client-control.md:51]: It seems that this PR uses `4 spaces` here, do we need to format it with `2 spaces`?

**Review by juzhiyuan — APPROVED** [review]

File: docs/zh/latest/plugins/client-control.md (hunk 2 above)
Comment (Xunzhuo) [reply docs/zh/latest/plugins/client-control.md:51]: Thanks but I think there`s no need to do this as other docs follow the format as well. 😊

**Pull request merged**